src/SmallStrainPlasticity/Elasticity.jl

```diff
@@ -0,0 +1,35 @@
+using Tensors
+abstract type AbstractElasticity{T} <:AbstractMaterial end
+# Linear isotropic elasticity
+struct LinearIsotropicElasticity{T} <:AbstractElasticity{T}
```

Comment:
Dont we already have a linear elastic material... maybe reuse it?

Reply:
Yes, I agree. 
Mostly we used slight differences so I didn't modify for this yet. But I also think a renaming would be required then to distinguish from e.g. LinearOrthotropicElasticity (that we could implement).